implementations/rust/ockam/ockam_command/src/pager.rs

```diff
@@ -36,8 +36,17 @@ fn paginate_with(pager_binary_path: PathBuf, help: clap::Error) -> miette::Resul
             .stdin
             .take()
             .expect("Failed to get pager's stdin");
+        // Check if the output is a terminal
```

Comment:
To avoid redundant information, let's write a single comment summarizing what's going on. I'd write something like:

```
// Strip ANSI escape sequences if stdout is not a TTY (e.g. when piping to another command)
```

Reply:
@adrianbenavides I updated my code. Please review it when you have time, thanks!